### PR TITLE
Update compatibility chart with Almalinux8 and remove CentOS8 since already deprecated

### PR DESCRIPTION
**Pull request by peterzhuamazon**
### Description
Tweak compatibility chart with Almalinux8 and remove CentOS8 since already deprecated

### Issues Resolved

Closes https://github.com/opensearch-project/opensearch-build/issues/4573

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
Thanks, @peterzhuamazon ! Should we close this PR? https://github.com/opensearch-project/documentation-website/pull/4980

**Comment by peterzhuamazon** [comment]
> Thanks, @peterzhuamazon ! Should we close this PR? #4980

Hi @hdhalter, closed old one and will update that once CentOS7 properly deprecated.

**Comment by hdhalter** [comment]
@peterzhuamazon - Will this be released with 2.13 or should it be backported to 2.12?

**Comment by peterzhuamazon** [comment]
> @peterzhuamazon - Will this be released with 2.13 or should it be backported to 2.12?

This can go with 2.13 and do not need to backport to 2.12 or before.

**Comment by hdhalter** [comment]
@Naarcha-AWS , @peterzhuamazon - Can we merge this into main?

**Comment by Naarcha-AWS** [comment]
@peterzhuamazon: Any update on the status of this one?

**Comment by hdhalter** [comment]
@Naarcha-AWS - This will not make 2.13 as per Peter. I removed the label.

**Comment by peterzhuamazon** [comment]
Hi,

@hdhalter @Naarcha-AWS this is ready to review again.

Thanks.

**Review by bbarani — APPROVED** [review]

File: _install-and-configure/install-opensearch/index.md [hunk 1]
```diff
@@ -17,15 +17,17 @@ This section details how to install OpenSearch on your host, including which ope
 
 ## Operating system compatibility
 
-OpenSearch and OpenSearch Dashboards are compatible with Red Hat Enterprise Linux (RHEL) and Debian-based Linux distributions that use [`systemd`](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, and Ubuntu Long-Term Support (LTS). While OpenSearch and OpenSearch Dashboards should work on most Linux distributions, we only test a subset. 
+OpenSearch and OpenSearch Dashboards are compatible with Red Hat Enterprise Linux (RHEL) and Debian-based Linux distributions that use [`systemd`](https://en.wikipedia.org/wiki/Systemd), such as Amazon Linux, and Ubuntu Long-Term Support (LTS). While OpenSearch and OpenSearch Dashboards should work on most Linux distributions, we only test a subset.
```
Comment (Naarcha-AWS) [inline _install-and-configure/install-opensearch/index.md:20]:
```suggestion
OpenSearch and OpenSearch Dashboards are compatible with Red Hat Enterprise Linux (RHEL) and Debian-based Linux distributions that use [`systemd`](https://en.wikipedia.org/wiki/Systemd), such as Amazon Linux, and Ubuntu Long-Term Support (LTS). While OpenSearch and OpenSearch Dashboards should work on most Linux distributions, the OpenSearch core team only tests a subset of distributions.
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

File: _install-and-configure/install-opensearch/index.md (hunk 1 above)
Comment (peterzhuamazon) [reply _install-and-configure/install-opensearch/index.md]: Thanks @Naarcha-AWS, updated.

**Comment by peterzhuamazon** [comment]
Thanks @Naarcha-AWS for the comment I have applied your suggestions.

Please review again

Thanks.

**Review by Naarcha-AWS — APPROVED** [review]

**Pull request merged**